perl-lib/OESS/lib/OESS/Database.pm

```diff
@@ -80,6 +80,7 @@ use GRNOC::Config;
 use OESS::Topology;
 use DateTime;
 use Data::Dumper;
+use Log::Log4perl;
```

Comment:
why did we add this?

Reply:
I added this before asking you about 'OESS.ACL' logging. I forgot to delete it.
Let me revert it.